examples/pytorch/MNIST/example2/README.md

```diff
@@ -0,0 +1,78 @@
+## MNIST example with MLFlow
+
+In this example, we train a model to predict handwritten digits.The autolog code uses Pytorch Lightning's MLFlowLogger to log metrics. 
+The code is trained using pytorch lightning loop and the autolog function call in the main - `autolog()`
+is responsible for logging the params, metrics, model summary and the model.
+This example logs metrics only after n epoch iterations. The iteration limit can be set in the autolog method using the parameter `log_every_n_iter=NUMBER-OF-ITERATIONS`.
+For ex: `autolog(log_every_n_iter=5)`
+
+### Code related to MLflow:
+* [`mlflow.pytorch.pytorch_autolog`]
+This is an experimental api that logs ML model artifacts and metrics.
+The metrics are logged during training of the model.
+
+## Setting tracking URI
```

Comment:
```suggestion
## Logging to a custom tracking server

To configure MLflow to log to a custom (non-default) tracking location, set the ``MLFLOW_TRACKING_URI`` environment variable, e.g. via  ``export MLFLOW_TRACKING_URI=http://localhost:5000/``.  For more details, see [the docs](https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded)
```

Could we reword this section as suggested? And move it to the end of the README? I think it'll only be relevant to a small number of readers